kitsune/search/v2/es7_utils.py

```diff
@@ -148,15 +151,19 @@ def index_objects_bulk(doc_type_name, obj_ids, timeout=settings.ES_BULK_DEFAULT_
     # if the request doesn't resolve within `timeout`,
     # sleep for `timeout` then try again up to `settings.ES_BULK_MAX_RETRIES` times,
     # before raising an exception:
-    es7_bulk(
-        es7_client(
-            timeout=timeout,
-            retry_on_timeout=True,
-            initial_backoff=timeout,
-            max_retries=settings.ES_BULK_MAX_RETRIES,
-        ),
-        (doc.to_action(action=action, is_bulk=True, **kwargs) for doc in docs),
-    )
+    try:
+        es7_bulk(
+            es7_client(
+                timeout=timeout,
+                retry_on_timeout=True,
+                initial_backoff=timeout,
+                max_retries=settings.ES_BULK_MAX_RETRIES,
+            ),
+            (doc.to_action(action=action, is_bulk=True, **kwargs) for doc in docs),
+        )
+    except BulkIndexError as e:
+        # This could be removed and completely silence this errors - need to re-evaluate
+        capture_exception(e)
```

Comment:
What's the advantage of using `sentry_sdk.capture_exception` over just re-raising it?

I don't think there's ever a situation where we want to completely silence `BulkIndexError` - I remember conflict errors are raised this way, and timeouts might also be raised with this error - both need reporting in sentry so we know to reindex the affected documents.

Is this for the case where we're deleting a document which doesn't exist in ES? It should be possible to parse the error to ignore those errors - they're raised in `/vendor/elasticsearch/helpers/actions.py#L188`